.github/workflows/ci.yaml

```diff
@@ -75,11 +76,17 @@ jobs:
 
     - name: Build and publish 
       env:            
-        DOCKERIMAGE: ${{ needs.check_image.outputs.imagetag }}
+        DOCKER_TAG_UBUNTU: ${{ needs.check_image.outputs.imagetag }}
       run: |
+        # The TOKEN and USERID are github secrets, Action failures at this step
+        # can come from a PR from a fork changing a file wichi forces a rebuild
+        # Resolve by making an internal PR of the Forked PR
         echo $DOCKER_TOKEN | docker login -u $DOCKER_USER --password-stdin
-        docker build . --file hip-clang.docker --tag $DOCKERIMAGE;
-        docker push $DOCKERIMAGE;
+
+        docker pull $DOCKER_IMAGE_UBUNTU:latest || true
```

Comment:
what `|| true` does here ? i haven't seen such thing before.

Reply:
That forces the return code to be success, there may not be a :latest tag and that would have caused an error.  It's ok if the :latest doesn't exist so just continue

Reply:
Looks like it pushes both tags 
docker push $DOCKER_IMAGE_UBUNTU:$DOCKER_TAG_UBUNTU;
docker push $DOCKER_IMAGE_UBUNTU:latest;